cmd/s3-gw/app.go

```diff
@@ -252,3 +255,22 @@ func getCacheOptions(v *viper.Viper, l *zap.Logger) *layer.CacheConfig {
 	}
 	return &cacheCfg
 }
+
+func getHandlerOptions(v *viper.Viper, l *zap.Logger) *handler.Config {
```

Comment:
Why you suddenly name it `Options` instead of `Config`?

Reply:
I thought it sounds better: the function reads options for handler and creates a config object. But if it seems non-logical I can rename it.

Reply:
It's okay, just wonder.